.github/workflows/wheel_image.yml

```diff
@@ -0,0 +1,72 @@
+name: Build custom Docker image for manylinux wheels
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_image:
+        description: 'The base Docker image to use'
+        default: 'manylinux_2_28'
+        type: choice
+        options:
+          - 'manylinux2014'
+          - 'manylinux_2_28'
+          - 'manylinux_2_34'
+
+      upload:
+        description: 'Whether to upload (push) the image to the container registry'
+        default: 'false'
+        required: true
+        type: 'boolean'
+
+      container_registry:
+        description: 'The name of the container registry to upload the image to (only useful if used with upload=true)'
+        default: 'ghcr.io'
+        required: true
+        options:
+          - 'ghcr.io'
+          - 'docker.io'
+
+      tag:
+        description: 'The tag for the final Docker image'
+        default: 'latest-'
```

Comment:
In light of #3306, the new images have the tag `manylinux_2_28`, so maybe this needs some rethinking.